examples/pytest/Readme.md

```diff
@@ -0,0 +1,26 @@
+# How to run this examples
+
+1. Clone the repository
+
+```bash
+git clone https://github.com/qase-tms/qase-python.git
+```
+
+2. Move to the directory with the examples
+
+```bash
+cd qase-python/examples/pytest
+```
+
+3. Install the required packages
+
+```bash
+pip install -r requirements.txt
+```
+
+4. Add the Qase token and project code to the `qase.config.json` file
```

Comment:
Using env variable is safer, because people won't commit their Qase token and leak it on GitHub. Let's promote safer practices.

Reply:
fixed